Dockerfile

```diff
@@ -39,7 +39,7 @@ COPY manage.py /manage.py
 
 COPY data/* /data/
 
-RUN echo yes | python manage.py collectstatic
+RUN echo yes | DJANGO_SECRET_KEY="collectstatic" python manage.py collectstatic
```

Comment:
So user need to update the key value inside the docker file? 

Is this step documented inside the README file?

Reply:
This is in there so the python collectstatic runs, it needs something set in the environment.

When you start up the docker file it will read from the .env file.

That's in the README (or the new sample env)